doc/api/n-api.md

```diff
@@ -239,11 +239,30 @@ available to the module code.
 
 ## Node-API version matrix
 
-Node-API versions are additive and versioned independently from Node.js.
-Version 4 is an extension to version 3 in that it has all of the APIs
-from version 3 with some additions. This means that it is not necessary
-to recompile for new versions of Node.js which are
-listed as supporting a later version.
+Up until version 9, Node-API versions are additive and versioned
+independently from Node.js. For example, version 4 is an extension to version
+3 in that it has all of the APIs from version 3 with some additions. Each
+Node.js version only supported a single Node-API version. For example
```

Comment:
nit: duplicated "For example"

Reply:
Also, I remember we had a push to get rid of the expression "For example", and just replace it with a concrete example.

Reply:
I removed the duplicate, I could not easily come up with something that made sense without "For example" though. @gabrielschulhof do you have a suggestion on that front?